src/definitions/modules/embed.js

```diff
@@ -677,7 +677,7 @@ $.fn.embed.settings = {
           src += '?' + parameters;
       }
       return ''
-        + '<iframe src="' + src + '"'
+        + '<iframe src="' + src.replace(/"/g,"") + '"'
```

Comment:
How about defining a helper?

Reply:
Implemented by fb905d2ec7f615e1c71f986691e6ab1208659669 (identical to dropdown)